test/integration/connectable/get-connection.js

```diff
@@ -39,8 +43,9 @@ describe('Connectable ::', function() {
         // Assert that a PG Client is returned
         assert(report.connection instanceof pg.Client);
 
-        // Assert that the connection has a release function
-        assert(report.connection.release);
+        // Assert that the connection has a release function, and call it to
+        // release the connection.
+        report.connection.release();
```

Comment:
Should we keep the `assert` here?

Reply:
Does it add value?  If `report.connection.release` is not defined, then this line will throw anyway.

Reply:
Fair point, but I'd like to keep this PR to just the minimal set of changes needed for the fix